coreteam.py

```diff
@@ -82,12 +102,34 @@
     'ps_sharebuttons': 'Share Buttons module',
     'ps_shoppingcart': 'Shopping cart module',
     'ps_socialfollow': 'Social Follow module',
-    'pscleaner': 'PS Cleaner module',
     'ps_themecusto': 'Theme customization',
     'ps_wirepayment': 'Wire payment module',
+    'pscleaner': 'PS Cleaner module',
     'psgdpr': 'PSGDPR',
+    'sekeywords': 'Search engine keywords',
+    'statsbestcategories': 'Best categories',
+    'statsbestcustomers': 'Best customers',
+    'statsbestmanufacturers': 'Best manufacturers',
+    'statsbestproducts': 'Best-selling products',
+    'statsbestsuppliers': 'Best suppliers',
+    'statsbestvouchers': 'Best vouchers',
+    'statscarrier': 'Carrier distribution',
+    'statscatalog': 'Catalog statistics',
+    'statscheckup': 'Catalog evaluation',
+    'statsdata': 'Data mining for statistics',
+    'statsequipment': 'Browsers and operating systems',
+    'statsforecast': 'Stats Dashboard',
     'statslive': 'Visitors online',
+    'statsnewsletter': 'Newsletter',
+    'statsorigin': 'Visitors origin',
+    'statspersonalinfos': 'Registered customer information',
+    'statsproduct': 'Product details',
     'statsregistrations': 'Registrations statistics modules',
+    'statssales': 'Sales and orders',
+    'statssearch': 'Shop search',
+    'statsstock': 'Available quantities',
+    'statsvisits': 'Visits and Visitors',
+    'welcome': 'OnBoarding',
```

Comment:
Please precise these are stats

Reply:
I actually used the module names in their repository's readme.md files. So, should I also update all the correspondant module names in each repository? What do you think?

Reply:
@PrestaShop/prestashop-product-team The name of project doesn't contain "Statistics" or "Stats". Is it wanted ?

Reply:
what is this about? 😅

Reply:
@sam-pires Module name for "stats*" doesn't indicate that there are about statistics.

Reply:
@sam-pires No answer to my question ?

Reply:
Hello 🙂 
Do we just need to change the name in this file or to update them everywhere the modules' names are mentioned?

If it is just here, we can add "xxx stats" at the end of their names
If it is everywhere, let's keep it that way for the moment